build.gradle

```diff
@@ -11,6 +11,6 @@ buildscript {
 }
 
 plugins {
-    id 'com.android.application' version '7.4.1' apply false
-    id 'com.android.library' version '7.4.1' apply false
+    id 'com.android.application' version '7.3.1' apply false
+    id 'com.android.library' version '7.3.1' apply false
 }
```

Comment:
Any reason why this was downgraded?

Reply:
i get build fail when i run the 7.4.1 version. i need to change that

Reply:
Ohh ok